.pre-commit-config.yaml

```diff
@@ -1,5 +1,37 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: 18.9b0
+  - repo: https://github.com/python/black
+    rev: stable
     hooks:
-    - id: black
+      - id: black
+  - repo: https://github.com/codespell-project/codespell
+    rev: v1.16.0
+    hooks:
+      - id: codespell
+        args:
+          - --quiet-level=2
+  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
This fails right now, and I don't want to make CI fail until we manage to refactor the functions. We should only include this along with other refactoring work so CI doesn't fail.